### PR TITLE
pyrush 1.0.8

### PR DESCRIPTION
**Pull request by skupr-anaconda**
License: https://github.com/jianlins/PyRuSH/blob/1.0.8/LICENSE
Requirements:
- https://github.com/jianlins/PyRuSH/blob/1.0.8/pyproject.toml
- https://github.com/jianlins/PyRuSH/blob/1.0.8/setup.py
- https://github.com/jianlins/PyRuSH/blob/1.0.8/requirements.txt

### CODE REVIEW

**Review by akabanovs — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,41 +21,49 @@ requirements:
...
 
 test:
   imports:
     - PyRuSH
-  commands:
-    - pip check
-  requires:
-    - pip
+  # Do not run pip check on the test environment, because of an error
+  # thinc 8.0.15 and spacy 3.3.1 has requirement pydantic!=1.8,!=1.8.1,<1.9.0,>=1.7.4, but you have pydantic 1.10.2.
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: would it be usable for customer then ?

Comment (skupr-anaconda) [reply recipe/meta.yaml]: You are right. I'll try to rebuild `thinc` and `spacy` and also update to the latest.

Comment (skupr-anaconda) [reply recipe/meta.yaml]: To fix the issue I'm updating spacy to 3.5.3 https://github.com/AnacondaRecipes/spacy-feedstock/pull/19

Comment (skupr-anaconda) [reply recipe/meta.yaml]: Fixed

**Review by ViridianMelody — APPROVED** [review]

**Review by varlackc — COMMENTED** [review]
We might want to remove the `abs.yaml` file before merging to defaults

**Pull request merged**